frontend/packages/kubevirt-plugin/src/selectors/vm-template/combined-dependent.ts

```diff
@@ -34,48 +34,46 @@ export const getFlavorLabel = (flavor: string) => {
   return undefined;
 };
 
-export const getOperatingSystems = (templates: TemplateKind[], userTemplate: string) => {
-  if (userTemplate) {
+export const getOperatingSystems = (templates: TemplateKind[], userTemplateName: string) => {
```

Comment:
same, why do we need to pass userTemplateName?

Reply:
@irosenzw please take a look. This should work the same way as getWorkloadProfiles.